flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/tasks/StreamTaskTest.java

```diff
@@ -1560,12 +1560,13 @@ public void testProcessWithUnAvailableInput() throws Exception {
                     waitingThread::start,
                     "Start WaitingThread after Task starts processing input.");
 
-            long startTs = System.currentTimeMillis();
-            throughputCalculator.calculateThroughput();
```

Comment:
is removing this line of code related to the bug? (I don't see how but it's confusing me a bit)

Reply:
I removed it because initially, this line started the inside timer but after the recent changes, incomingDataSize can do it as well so right now this line doesn't make sense(but it also doesn't change any logic in test).

Reply:
In that case can you extract this to a separate hotfix commit? Mixing bug fixes and refactoring is very confusing for reviewing and later reading the code.